app/assets/stylesheets/application/header.scss

```diff
@@ -194,3 +194,7 @@
     padding: 1px 0 4px;
   }
 }
+
+.hide_me{
```

Comment:
can we rename this class to maybe `dim`? or something like that? it's not really for hiding, it's for changing the opacity slightly.